opencti-platform/opencti-graphql/src/utils/confidence-level.ts

```diff
@@ -105,7 +111,10 @@ export const controlUpsertInputWithUserConfidence = <T extends ObjectWithConfide
     throw LockTimeoutError({ user_id: user.id, element_id: existingElement.id }, 'User has no effective max confidence level and cannot upsert this element');
   }
   const userMaxConfidence = user.effective_confidence_level?.max_confidence as number;
-  const confidenceLevelToApply = capInputConfidenceWithUserMaxConfidence(userMaxConfidence, inputElementOrPatch.confidence);
+  const override = user.effective_confidence_level?.overrides?.find((e) => e.entity_type === existingElement.entity_type);
+  const overrideMaxConfidence = override?.max_confidence ?? 0;
+  const maxConfidenceForEntity = Math.max(userMaxConfidence, overrideMaxConfidence);
```

Comment:
Now that I think again, this seems wrong.
* If a user has 50CL, but 80 on Report, it works
* If a user has 80 CL, but 30 on Report, the maxConfidenceForEntity is 80 due to the math.max. 

That's not how overrides shall work ; if an override is defined for this type of entity, we use this value plain and simple.